stripe/src/main/java/com/stripe/android/view/PaymentAuthWebView.kt

```diff
@@ -138,7 +142,8 @@ internal class PaymentAuthWebView @JvmOverloads constructor(
 
         private fun hideProgressBar() {
             logger.debug("PaymentAuthWebViewClient#hideProgressBar()")
-            progressBar.isGone = true
+
+            isPageLoaded.value = true
```

Comment:
`isPageLoaded.postValue(true)`

Reply:
`postValue` is only necessary when not on the main thread.